### PR TITLE
adds hotkeys to parser output

### PR DESCRIPTION
**Pull request by DavidMente**
double checked with julas output. other than julas I think it would be preferable to map the hotkeys to the keys that are actually used (julas uses "0" for "1" etc)

### CODE REVIEW

File: src/Player.ts [hunk 1]
```diff
@@ -130,6 +134,18 @@ class Player {
       selecthotkey: 0,
       esc: 0,
     };
+    this.hotkeys = {
```
Comment (PBug90) [inline src/Player.ts:137]: Can we rename that variable to groupHotkeys so we know it is just about the group assignment/usage?

File: test/replays/132/replays.test.ts [hunk 2]
```diff
@@ -141,3 +141,11 @@ it("parses 1.32.8 replay with observer on defeat setting", async () => {
   );
   expect(test.settings.observerMode).toBe("ON_DEFEAT");
 });
+
+it("should parse hotkeys correctly", async () => {
+  const test = await Parser.parse(path.resolve(__dirname, "reforged1.w3g"));
+  expect(test.players[0].hotkeys[1]).toEqual({ assigned: 1, used: 29 });
```
Comment (PBug90) [inline test/replays/132/replays.test.ts:147]: Please add the new property to the jsonschema aswell.

**Review by PBug90 — CHANGES_REQUESTED** [review]

File: test/replays/132/replays.test.ts (hunk 2 above)
Comment (DavidMente) [reply test/replays/132/replays.test.ts:147]: Do you do that manually?

Comment (PBug90) [reply test/replays/132/replays.test.ts]: Yes, the JSONSchema is a manually edited document and the output is asserted to match it in a unit test.

**Comment by PBug90** [comment]
Also please remove the .idea subdirectory.

File: test/schema.json [hunk 3]
```diff
@@ -397,6 +397,11 @@
                 }
               }
             }
+          },
+          "groupHotkeys": {
```
Comment (PBug90) [inline test/schema.json:401]: I am aware that the schema currently isnt very strict with properties in most of its parts, but I am planning on updating it in the future. The PR looks good so far, so the only change needed is to add explicit property keys for the groupHotkeys object with their types and to disallow additional keys to be present.

**Review by PBug90 — CHANGES_REQUESTED** [review]

File: test/schema.json (hunk 3 above)
Comment (PBug90) [reply test/schema.json:401]: Going to merge it anyway today and update the schema myself afterwards to get the release going with all the pending changes. Thank you for adding the feature!

**Review by PBug90 — APPROVED** [review]

**Pull request merged**